APISpec.md

```diff
@@ -118,9 +115,7 @@ Adopts a creature. User's affinity level with the specified creature must be 100
 **Response:**
 
 ```json
-{
-    "success": "boolean"
-}
+"OK"

```

Comment:
and how do we represent a simple 200 status code in the APISpec?

Reply:
Based on what I've seen, it would be represented as {"content": "string", "status_code": integer}. And the actual functions would return a JSONResponse, as done in server.py. ex. `return JSONResponse("response", status_code=422)`